fe/src/assets/css/tailwind.css

```diff
@@ -29,3 +29,62 @@ body {
   white-space: nowrap;
   width: 100%;
 }
+/* Paginator */
+.paginator-arrow-heads {
+  @apply flex items-center;
+}
+.paginator-icon {
+  @apply items-center justify-center rounded-md border border-transparent leading-tight;
+}
+.paginator-icon.disabled {
+  @apply cursor-not-allowed text-neutral-500;
+}
+.paginator-icon.enabled {
+  @apply cursor-pointer hover:text-primary-10;
+}
+.paginator-icon.arrow {
+  @apply ml-1 flex h-4 w-4;
+}
+.paginator-icon.arrow .icon {
+  @apply h-full w-full;
+}
+.paginator-icon.ellipsis {
+  @apply ml-2 flex h-6 w-6;
+}
+
+.paginator-number {
+  @apply ml-2 flex h-6 w-6 cursor-pointer items-center justify-center rounded-md border border-transparent text-center text-xs leading-tight  hover:border-primary-10;
+}
+.paginator-number.active {
+  @apply bg-primary-10 text-white;
+}
+.paginator-number.not-active {
+  @apply text-neutral-500 hover:text-primary-10;
+}
+/* End of Paginator CSS */
+
+/* Campaign Card */
+.campaign-card {
+  @apply w-85 h-100.5 rounded-2xl overflow-hidden bg-white shadow-card;
+}
+.campaign-card .header {
+  @apply h-42.5 w-full object-cover;
+}
+.campaign-card .body {
+  @apply mx-4 mt-4 leading-[150%];
+}
+.campaign-card .body .title {
+  @apply text-dark text-base font-bold;
+}
+.campaign-card .body .description {
+  @apply mt-2 text-disabled text-sm font-normal flex items-center;
+}
+.campaign-card .body .stats {
+  @apply mt-6 w-71.75;
+}
+.campaign-card .body .stats .values {
+  @apply font-bold text-dark flex justify-between;
+}
+.campaign-card .body .stats .labels {
+  @apply font-normal text-primary-500 flex text-sm justify-between;
+}
```

Comment:
Transfer these styles to the campaign card component.
No need to create custom class here. Just add the class directly to the elements.

```suggestion

```